### DISCUSSION TITLE
容器集群日志无法采集到

### DISCUSSION THREAD

**Discussion by hanzoi** [alibaba/ilogtail#1244] [Help]
集群信息：AWS EKS 集群 （单节点）

以下是 ilogtail-user-cm 配置：
enable: true
inputs:
  - Type: file_log
    LogPath: /var/log
    FilePattern: "*.log"
    ContainerFile: true
flushers:
  - Type: flusher_stdout
    OnlyStdout: true

目前在 ilogtail 容器中可以搜到其他容器中的 .log 后缀日志，但是就是采集不到
![image](https://github.com/alibaba/ilogtail/assets/61114924/6f517d11-2d6e-47ed-9760-609390b2ddf9)

以下是 ilogtail.LOG 日志：
<img width="1441" alt="image" src="https://github.com/alibaba/ilogtail/assets/61114924/44b25c6e-63de-41ec-954a-3b539a7f99ce">
logger_initialization.log ：
<img width="834" alt="image" src="https://github.com/alibaba/ilogtail/assets/61114924/9e9ae1ae-dff1-4fed-8371-1f10ae650189">
在 logtail_plugin.LOG 有看到如下报错，但是不确定是否有什么影响
<img width="1502" alt="image" src="https://github.com/alibaba/ilogtail/assets/61114924/b3296123-75ab-4305-90bd-b20fc5a664ef">

**Comment by hanzoi** [comment]
这个是标准输出中看到的东西
<img width="1313" alt="image" src="https://github.com/alibaba/ilogtail/assets/61114924/b252d9f7-c6ce-4f83-bed4-3241b2442fff">

**Comment by hanzoi** [comment]
解决了 我是containerd ，需要加这两个变量

**Comment by hanzoi** [comment]
![image](https://github.com/alibaba/ilogtail/assets/61114924/65acc8f2-234a-48bf-8d3b-b9e8a81f5819)

**Comment by zhangrr** [comment]
加了也还是不行，eks集群
